enclave-manager/web/src/components/enclaves/logs/LogViewer.tsx

```diff
@@ -247,7 +249,7 @@ const SearchControls = ({ searchState, onChangeSearchState, logLines }: SearchCo
     [logLines, onChangeSearchState],
   );
 
-  const debouncedUpdateMatches = useMemo(() => debounce(updateMatches, 100), [updateMatches]);
+  const debouncedUpdateMatches = useMemo(() => debounce(updateMatches, 300), [updateMatches]);
```

Comment:
@Dartoxian I would prefer a quicker update, but wondering if you hit something that made 100ms infeasible. Let me know. Else I think faster updating, even as you type is a better experience

Reply:
yep I was finding that with the 100ms speed I was encountering a set state limit error with react. Maybe what would be more appropriate here, and would feel more responsive would be a `throttle` rather than a `debounce`. I'll implement that on Thursday.

Reply:
ok makes sense. Thanks!

Reply:
`throttle` is working a lot better than `debounce` in this situation